README.md

```diff
@@ -1 +1,88 @@
-# BloomDiscordBot
\ No newline at end of file
+# BloomDiscordBot
+
+
+# Emotes:
+
+When a mapped contributors Emoji is used in Blooms Discord server, the contributor will be DM'd advising they have been mentioned in a chat.
+A link to the message they have been mentioned in will be included. 
+
+# Commands:
+
+The following commands can be used to manage, and list contributors. All commands 
+start with: $
+
+**Remove Contributor:** 
+
+!removecontributor <uid> 
+
+**Example:**
+
+```
+$removecontributor @contributor
+```
+
+**Response:**
+```
+Contributor Sarahtonein removed successfully!
+```
+
+**Add Contributor:**
+```
+$addcontributor
+```
+
+```
+1. $addcontributor
+
+2. Bot will respond with required details.
+
+3. Respond with required details and the contributor will be added.
+
+```
+
+**Example:**
+
+```
+$addcontributor
+```
+
+```
+To add a contributor, please provide the following information:
+Name
+User ID (UID)
+Emoji ID
```

Comment:
How are "normal" users getting the ID and the Emoji ID?
I can see that being very challenging and at a minimum laborious

Reply:
> How are "normal" users getting the ID and the Emoji ID? I can see that being very challenging and at a minimum laborious

Documentation yet to be updated after Lap provided feedback when testing the bot. Wording here was also slightly off. When adding a contributor you can provide the :emoji: and the bot will identify the Emoji ID.